index.html

```diff
@@ -28,6 +28,7 @@ <h1 id ="title">freeCodeCamp Survey Form</h1>
             <option>prefer not to say</option>
             <option>other</option>
         </select><br>
+        
         Would you recommend freeCodeCamp to a friend?<br>
```

Comment:
- [ ] Please try to be more professional in your comments and add solid reasons why student need to use capitalize for the heading and how it make an independent sentence, also it seems like you are ordering, your comment should be clear, polite and professional. Kindly look at the point below, which will help you perform code reviews very well.
   - Kindly remove the spacing or indentation because it actually makes your code looks messy and makes your code base setup unprofessional.   

## Your comment

>  Capitalize the first letters for this heading since it is an independent sentence.